.github/workflows/build.yml

```diff
@@ -110,6 +123,13 @@ jobs:
             /usr/share/dotnet \
             /usr/share/swift \
             &
+      - name: Set env for self-hosted runners
```

Comment:
these should be pre-set in the runner environment. in the incus runners, I add worker environment to `.env` in the runner folder - is that possible with the k3s workers?

Reply:
> the `SCCACHE_GHA_ENABLED` env variable is set in the workflow

maybe make that conditional on `self-hosted==false` instead?